build.sh

```diff
@@ -17,16 +17,16 @@ function emsdk_setup() {
 
     cd emsdk
     ./emsdk update
-    ./emsdk install sdk-fastcomp-1.38.30-64bit
-    ./emsdk activate sdk-fastcomp-1.38.30-64bit
+    ./emsdk install 1.39.6
+    ./emsdk activate 1.39.6
     cd ..
 
     # u6 customizations:
     EMSDK="`pwd`/emsdk"
 }
 
 # Host tools required by Android, WebGL, and iOS builds
-MOBILE_HOST_TOOLS="matc matinfo resgen cmgen"
+MOBILE_HOST_TOOLS="matc resgen cmgen"
```

Comment:
we were using matinfo to extract material data for new renderer - not sure if we'll still want to keep it around

Reply:
We build desktop mac separately now which already has `matinfo`. We don't need to build it separate when building for mobile. I removed it.